includes/deprecated/deprecated-functions.php

```diff
@@ -1170,3 +1170,21 @@ static function ( $url ) {
 		$list
 	);
 }
+
+/**
+ * This function will help you load Stripe SDK based on the conditions.
+ *
+ * @since 2.5.5
+ *
+ * @deprecated  @unreleased
+ * @return void
+ */
+function give_stripe_load_stripe_sdk() {
+	$stripe_sdk_compatibility = give_get_option( 'stripe_sdk_incompatibility', 'composer' );
+
+	if ( 'composer' === $stripe_sdk_compatibility ) {
+		require_once GIVE_PLUGIN_DIR . 'vendor/autoload.php';
+	} elseif ( 'manual' === $stripe_sdk_compatibility ) {
+		require_once GIVE_PLUGIN_DIR . 'vendor/stripe/stripe-php/init.php';
+	}
+}
```

Comment:
@ravinderk What's the point of doing this? It's an internal function, right? Not something we encourage 3rd parties or our other add-ons to use?

Reply:
@JasonTheAdams Yes, I agree with you. It intended to use internal but yet it is a public function. I can remove it completed but I thought that this can be a good way to deprecated this.

Reply:
@JasonTheAdams I think over this and i will remove this

Reply:
fixed: https://github.com/impress-org/givewp/commit/1010bcc6c86b42a8e8d43c2d96410bb4cdb6ce64